### DISCUSSION TITLE
Auto generated field value from sibling field hooks not updating the field value on the admin UI, need refresh page

### DISCUSSION THREAD

**Discussion by dollarplanet** [payloadcms/payload#9988] [Q&A]
Just like the title says, I have created a "before Change" hook to generate sibling field values. The values ​​are successfully saved to the database but it requires a page refresh to see the latest sibling field values ​​in the admin UI.

Is it possible to force the admin UI to get the latest data via field hooks? I was thinking about revalidatePath or something similar.

This is my beforeChange hooks look like
```
async ({ value, siblingData}) => {

// generate value logic

siblingData.fieldName = generated;

return value;

}
```

**Comment by bhofstaetter** [comment]
could you find a solution for this that doesn't rely on a custom fiel d?
